devel/bazelisk/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/bazelbuild/bazelisk 1.11.0 v
+revision            0
+categories          devel
+maintainers         {@0xIDANT danieltrautmann.com:me} openmaintainer
+license             Apache-2
+
+description         A user-friendly launcher for Bazel
+
+long_description    Bazelisk is a wrapper for Bazel written in Go. \
+                    It automatically picks a good version of Bazel given your \
+                    current working directory, downloads it from the official \
+                    server (if required) and then transparently passes through \
+                    all command-line arguments to the real Bazel binary. You \
+                    can call it just like you would call Bazel.
+
+conflicts           bazel
+
+fetch.type          git
+build.cmd           ./build.sh
+build.env-append    GOPROXY=https://proxy.golang.org,direct
+
+destroot {
+    xinstall -m 0755 \
+        ${worksrcpath}/bin/${name}-${goos}-${goarch} \
+        ${destroot}${prefix}/bin/${name}
+
+    ln -s ${prefix}/bin/${name} ${destroot}${prefix}/bin/bazel
+}
+
+checksums           ${distname}${extract.suffix} \
+                        rmd160  c27d46e18702ee16735d31818f931b93189c1d6c \
+                        sha256  0842cb7c0e828839e75c80c253bb0ba7a1e95120e21567e86f5bfb60ee59e467 \
+                        size    152999
```

Comment:
You specify checksums for a main distfile, but you also specify `fetch.type git` which means a main distfile won't be used.

Why do you need `fetch.type git`?

Reply:
I used `fetch.type git` because the build process needs a Git repository to get information about the version. I guess I could circumvent this by patching some files.

Reply:
One more thing.  Since you are now using `build.env-delete    GO111MODULE=off GOPROXY=off`, you no longer are using `go.vendors`.

Remove the entire `go.vendors` section, and your `checksums` can just look like this:
```
checksums               rmd160  c27d46e18702ee16735d31818f931b93189c1d6c \
                        sha256  0842cb7c0e828839e75c80c253bb0ba7a1e95120e21567e86f5bfb60ee59e467 \
                        size    152999
```

Reply:
Thanks, done